BigQueryMigration/.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+{"version":1,"defects":[],"times":{"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::createMigrationWorkflowTest":0.021,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::createMigrationWorkflowExceptionTest":0.001,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::deleteMigrationWorkflowTest":0.001,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::deleteMigrationWorkflowExceptionTest":0,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::getMigrationSubtaskTest":0.001,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::getMigrationSubtaskExceptionTest":0,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::getMigrationWorkflowTest":0,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::getMigrationWorkflowExceptionTest":0,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::listMigrationSubtasksTest":0.001,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::listMigrationSubtasksExceptionTest":0,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::listMigrationWorkflowsTest":0.001,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::listMigrationWorkflowsExceptionTest":0,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::startMigrationWorkflowTest":0.001,"Google\\Cloud\\BigQuery\\Migration\\Tests\\Unit\\V2\\MigrationServiceClientTest::startMigrationWorkflowExceptionTest":0}}
```

Comment:
we should delete this! I added this to `.gitignore` [here](https://github.com/googleapis/google-cloud-php/pull/5542/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R10)

Reply:
Thanks! I just noticed this, too, while generating another client - fixed now!